src/simplejavacalculatorTest/CalculatorTest.java

```diff
@@ -14,69 +14,69 @@
 	void calculateBiNormalTest() {
 		Calculator calculator = new Calculator();
 		calculator.calculateBi(Calculator.BiOperatorModes.normal, 2.0);
-		Assertions.assertEquals(NaN, calculator.calculateBi(Calculator.BiOperatorModes.normal, 3.0));
+		Assertions.assertEquals(NaN, calculator.calculateBi(Calculator.BiOperatorModes.normal, 3.0),0.0);
     }
 	
 	@Test
 	void calculateBiAddTest() {
 		Calculator calculator = new Calculator();
 		calculator.calculateBi(Calculator.BiOperatorModes.add, 3.0);
-		Assertions.assertEquals(5.5, calculator.calculateBi(Calculator.BiOperatorModes.normal, 2.5));
+		Assertions.assertEquals(5.5, calculator.calculateBi(Calculator.BiOperatorModes.normal, 2.5),0.0);
 	}
 	
 	@Test
 	void calculateBiMinusTest() {
 		Calculator calculator = new Calculator();
 		calculator.calculateBi(Calculator.BiOperatorModes.minus, 3.1415);

```

Comment:
Our static analysis engine detects that your pull request includes a change that uses the rough value of Math.PI. According to the [best practice](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#cnt-rough-value-of-known-constant-found-cnt-rough-constant-value), it's recommended to use the predefined library constant for code clarity and better precision.